### PR TITLE
Update java-testing.md

### PR DESCRIPTION
**Pull request by jdneo**

### CODE REVIEW

**Comment by jdneo** [comment]
@nickzhums

**Review by nickzhums — APPROVED** [review]

File: docs/java/java-testing.md [hunk 1]
```diff
@@ -169,6 +169,9 @@ Currently the supported configurations are:
 - **sourcePaths**: Specify the extra source paths when debugging the tests.
 - **vmArgs**: Specify the extra options and system properties for the JVM.
 - **workingDirectory**: Specify the working directory when running the tests.
+- **testKind**: Specify the targeting test framework for this test configuration. Supported values are `junit`, `testng`.
+- **filters**: Specify the test filters.
+  - **tags**: Specify the tags to be included or excluded. Tags having `!` as the prefix will be **excluded**. Note: This setting only takes effect when `testKind` is set to `junit`
```
Comment (gregvanl) [inline docs/java/java-testing.md:174]: Did you mean for "tags" to be indented one level?

image

Comment (jdneo) [reply docs/java/java-testing.md:174]: Yes, it belongs to `filters`.

The setting looks like in this way

```
...
filters: {
    tags: ["foo", "!bar"]
}
...
```

**Review by gregvanl — APPROVED** [review]
Thanks!
If topics have had a full review and update, you can bump the DateApproved metadata value to the current date to show freshness.
I'll update this topics value with a follow up PR.

**Pull request merged**